packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -153,6 +153,7 @@ export class Compass {
   }
 
   async recordLogs(): Promise<void> {
+    debug('Setting up renderer log listeners ...');
```

Comment:
This was just helpful to understand that it's not trying to get the puppeteer connection that fails, but actually starting the remote webdriver session